test/data/README.md

```diff
@@ -0,0 +1,43 @@
+# Data Test README
+
+## spreadsheets-health.test.ts
+
+The main test in this file ensures that registered spreadsheets are in sync with data that's checked in. This allows us to have spreadsheets that don't incur significant tech debt every time we need to convert them to JSON (or vice versa).
+
+The easiest mental model for what the test is doing is making sure the spreadsheet and JSON data are the same.
+
+More specifically every row in a registered spreadsheet must be in sync with checked-in JSON records in the repo. A row can:
+
+1. fail CollectedIncentive validation. This means it doesn't even meet our schema for collected data in spreadsheet form. These go in the `incentives_invalid_collected.json` file for that state.
+2. fail StateIncentive validation. These records are valid spreadsheet records but something is failing in the process of converting them to servable JSON. These go in the `incentives_invalid_state.json` file for that state.
+3. be valid, servable JSON. These go in the `incentives.json` file for that state.
+
+Eventually, we'd want all spreadsheet records go in category #3, but we're not there yet. In the meantime, categories #1 and #2 are for meeting the spreadsheets as they are and making sure they don't get worse (worse meaning more out of sync from our schemas and data).
+
+### Registering a state and fixing failures the first time
+
+To add a new state, flip the `runSpreadsheetHealthCheck` boolean in `scripts/incentive-spreadsheet-registry.ts` to true. Then run the test. It'll almost certainly fail.
+
+Assuming it does, run the `incentive-spreadsheet-to-json.ts` script (documented in `scripts/README.md`) to overwrite the existing JSON files, and then look at the diffs to see what needs to be changed. A diff means the spreadsheet and checked-in data are out of sync, and since the checked-in data is automated downstream processing of the spreadsheet, it's likely the spreadsheet that needs to be updated.
```

Comment:
This is saying to look at the diff between the original JSON file(s) and the generated ones, correct? 

I think that something to think of here is that it is possible someone has updated/added rows to the spreadsheet, and that could be why this is failing as well. I'm not sure if it is worth calling out that the spreadsheet is likely to need updating vs referring to the "potential reasons" section below.

Reply:
Ack. I pretty much agree with all of this and will make some changes to reflect that.